security/tests/integration/cmd/main.go

```diff
@@ -0,0 +1,275 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"istio.io/istio/security/pkg/pki/ca/controller"
+	"istio.io/istio/security/tests/integration"
+	"istio.io/istio/tests/integration/framework"
+)
+
+const (
+	testID = "istio_ca_secret_test"
+	// Specifies how long we wait before a secret becomes existent.
+	secretWaitTime = 20 * time.Second
+	// Certificates validation retry
+	certValidateRetry      = 10
+	certValidationInterval = 1 // Initially wait for 1 second. This value will be increased exponentially on retry
```

Comment:
What about putting the trailing comment on top to be consistent with previous ones?